ginga/Control.py

```diff
@@ -1466,8 +1477,8 @@ def get_loaded_image(self, imname):
     def add_image(self, image, silent=False, bulk_add=False):
 
         imname = image.get('name', None)
-        assert imname is not None, \
-               ValueError("image has no name")
+        if imname is None:
+            raise ValueError("image has no name")
 
```

Comment:
I think `assert` is fine if you want `AssertionError`. But here, you are reraising the error as `ValueError`, so it is quite confusing to read. But I guess it comes down to personal preference. Do you want me to change it back?